.gitignore

```diff
@@ -150,3 +150,4 @@ packaging/deb/freerdp-nightly/freerdp-nightly-dbg
 
 # VisualStudio Code
 .vscode
+cache/
```

Comment:
please remove.

Reply:
This is the directory where the script/android-build-freerdp.sh puts the archives it downloads from github for re-use.

Reply:
@wintersandroid ah, ok, point taken.